lib/lookup.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
-const normgit = require('normalize-git-url');
-const util = require('util');
 const path = require('path');
+const util = require('util');
+
+const normgit = require('normalize-git-url');
```

Comment:
hmm okay yeah. Can you perhaps squash the commits? I'm not sure if this needs to be 3 different commits?